Sources/GEOSwift/GEOS/GeometryConvertible+GEOS.swift

```diff
@@ -425,6 +425,22 @@ public extension GeometryConvertible {
         }
     }
 
+    func bufferWithStyle(width: Double, quadsegs: Int32 = 8, endCapStyle: Int32 = 1, joinStyle: Int32 = 1, mitreLimit: Double = 5.0) throws -> Geometry? {
```

Comment:
What's the rationale for these specific default parameter values?

Reply:
it's default values used in geos